cmd/buildah/common.go

```diff
@@ -112,6 +124,50 @@ func getStore(c *cobra.Command) (storage.Store, error) {
 	return store, err
 }
 
+// GetRootlessRuntimeDir returns the runtime directory when running as non root
+func GetRootlessRuntimeDir() (string, error) {
```

Comment:
can't we use it from containers/storage?

Reply:
can't. GetRootlessRuntimeDir() in container/storage returns `/tmp/uid` instead of `/run/user/uid` ... :thinking:

Reply:
`GetRootlessRuntimeDir()` returns  `os.TempDir()/rootlessUid` because current os.Getenv("XDG_RUNTIME_DIR") != ""
in my machine `os.Getenv("XDG_RUNTIME_DIR")` is already `/run/user/uid` before executing `GetRootlessRuntimeDir().`
@giuseppe  Why we need to set XDG_RUNTIME_DIR here? https://github.com/containers/buildah/blob/316bd0a68b20324404304dfc731065a815f16593/cmd/buildah/common.go#L34